code/game/objects/items/weapons/handcuffs.dm

```diff
@@ -85,45 +85,70 @@
 	icon_state = "sinewcuff"
 	item_state = "sinewcuff"
 	belt_icon = null
-	breakouttime = 300 //Deciseconds = 30s
+	breakouttime = 30 SECONDS
 	cuffsound = 'sound/weapons/cablecuff.ogg'
 
 /obj/item/restraints/handcuffs/cable
 	name = "cable restraints"
 	desc = "Looks like some cables tied together. Could be used to tie something up."
 	icon_state = "cuff_white"
+	item_state = "cuff_white"
 	belt_icon = null
 	origin_tech = "engineering=2"
 	materials = list(MAT_METAL=150, MAT_GLASS=75)
-	breakouttime = 300 //Deciseconds = 30s
+	breakouttime = 30 SECONDS
 	cuffsound = 'sound/weapons/cablecuff.ogg'
 
 /obj/item/restraints/handcuffs/cable/red
 	color = COLOR_RED
+	item_state = "cuff_red"
 
 /obj/item/restraints/handcuffs/cable/yellow
 	color = COLOR_YELLOW
+	item_state = "cuff_yellow"
 
 /obj/item/restraints/handcuffs/cable/blue
 	color = COLOR_BLUE
+	item_state = "cuff_blue"
 
 /obj/item/restraints/handcuffs/cable/green
 	color = COLOR_GREEN
+	item_state = "cuff_green"
 
 /obj/item/restraints/handcuffs/cable/pink
 	color = COLOR_PINK
+	item_state = "cuff_pink"
 
 /obj/item/restraints/handcuffs/cable/orange
 	color = COLOR_ORANGE
+	item_state = "cuff_orange"
 
 /obj/item/restraints/handcuffs/cable/cyan
 	color = COLOR_CYAN
+	item_state = "cuff_cyan"
 
 /obj/item/restraints/handcuffs/cable/white
 	color = COLOR_WHITE
 
 /obj/item/restraints/handcuffs/cable/random/New()
-	color = pick(COLOR_RED, COLOR_BLUE, COLOR_GREEN, COLOR_WHITE, COLOR_PINK, COLOR_YELLOW, COLOR_CYAN)
+	color = pick(COLOR_RED, COLOR_BLUE, COLOR_GREEN, COLOR_WHITE, COLOR_PINK, COLOR_YELLOW, COLOR_CYAN, COLOR_ORANGE)
+	switch(color)
+		if(COLOR_RED)
+			item_state = "cuff_red"
+		if(COLOR_BLUE)
+			item_state = "cuff_blue"
+		if(COLOR_GREEN)
+			item_state = "cuff_green"
+		if(COLOR_WHITE)
+			item_state = "cuff_white"
+		if(COLOR_PINK)
+			item_state = "cuff_pink"
+		if(COLOR_YELLOW)
+			item_state = "cuff_yellow"
+		if(COLOR_CYAN)
+			item_state = "cuff_cyan"
+		if(COLOR_ORANGE)
+			item_state = "cuff_orange"
```

Comment:
instead of doing this, why don't you delete these sprites, and make it use the colour variable? 
in `update_inv_r_hand()` (and left hand) change 
```
			standing = mutable_appearance(r_hand.sprite_sheets_inhand[dna.species.name], "[t_state]", layer = -R_HAND_LAYER)
		else
			standing = mutable_appearance(r_hand.righthand_file, "[t_state]", layer = -R_HAND_LAYER)
```
with 
```
			standing = mutable_appearance(r_hand.sprite_sheets_inhand[dna.species.name], "[t_state]", layer = -R_HAND_LAYER,  color = r_hand.color)
		else
			standing = mutable_appearance(r_hand.righthand_file, "[t_state]", layer = -R_HAND_LAYER, color = r_hand.color)
```

that way you can pass the recolouring to inhands.

Reply:
I like this idea, but I am concerned it may conflict with other in hands that have colors (such as normal cable coils), but it will probably be worth it and not to bad to change/fix. I am slightly concerned with any fixes required by this change that they might conflict with #18303 since they both touch cablecuffs/cables in a similar way.
Anyways, this change will be in the next commit.

Reply:
Test it, and see how it looks with other inhands like cable coils. Slime potion hardsuits.